src/shared/Z.EF.Plus.BatchDelete.Shared/BatchDeleteVisitor.cs

```diff
@@ -54,5 +54,15 @@ protected override Expression VisitMethodCall(MethodCallExpression node)
             }
             return base.VisitMethodCall(node);
         }
+
+        protected override Expression VisitMember(MemberExpression node)
+        {
+            if (node.Expression?.GetType().Name == "PropertyExpression")
```

Comment:
Why don't we use "is" type comparison?

Reply:
As @sshukurov mentioned already, the PropertyExpression type is internal

Reply:
Sorry, missed that)